dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -987,6 +971,35 @@ public void saveAction(final WorkflowAction action) throws DotDataException,Alre
 
 	}
 
+	private List<WorkflowStep> findProxiesSteps(final WorkflowAction action) throws DotDataException {
+
+		final ImmutableList.Builder<WorkflowStep> stepsBuilder =
+				new ImmutableList.Builder<>();
+
+		final List<Map<String, Object>> stepIdList =
+				new DotConnect().setSQL(sql.SELECT_STEPS_ID_BY_ACTION)

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)